engines/bastion_katello/test/organizations/fenced-pages.service.test.js

```diff
@@ -6,12 +6,12 @@ describe('Factory: FencedPages', function() {
     }));
 
     it("should list all the fenced pages", function () {
-        expect(FencedPages.list().length).toBe(10);
+        expect(FencedPages.list().length).toBe(11);
```

Comment:
What is the purpose of this test? What kind of bugs are we expecting it to catch?